modules/update-duration-mco.adoc

```diff
@@ -50,4 +50,4 @@ ip-10-0-207-224.us-east-2.compute.internal  Ready                       worker
 +
 If the status of the node is `NotReady` or `SchedulingDisabled`, then the node is not available and this impacts the update duration.
 +
-You can check the status of nodes from the *Administrator* perspective in the web console by expanding **Compute** → **Node**.
+You can check the status of nodes from the *Administrator* perspective in the web console by expanding **Compute** **>** **Nodes**.
```

Comment:
Per [our docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference), this is actually the way you'd want to make the arrow icon:
```suggestion
You can check the status of nodes from the *Administrator* perspective in the web console by expanding **Compute** -> **Nodes**.
```

Sorry to make you go back and forth on this 😅

Reply:
lol no prob! ty :)